plugins/inputs/gnmi/gnmi.go

```diff
@@ -52,6 +52,7 @@ type GNMI struct {
 	Subscriptions    []Subscription    `toml:"subscription"`
 	TagSubscriptions []TagSubscription `toml:"tag_subscription"`
 	Aliases          map[string]string `toml:"aliases"`
+	MaxMsgSize       int               `toml:"max_msg_size"`
```

Comment:
Can we please make this a `config.Size` so people can specify something like `8MB`?